docs/MANPAGE_SYNTAX.md

```diff
@@ -111,9 +112,20 @@ $ cat $HOME/Dockerfile | podman command --option
 
 Description of the EXAMPLE two
 ```
-$ podman command --redhat
+# podman command --redhat
```

Comment:
Please remove the name redhat and just make it --option.  This is an Open Source project not a `redhat` project.

Reply:
Done.